jest.config.js

```diff
@@ -1,3 +1,9 @@
 module.exports = {
-  setupFiles: ["<rootDir>/test/setup.js"]
+  moduleFileExtensions: ["js", "jsx", "json", "styl"],
+  setupFiles: ["<rootDir>/test/setup.js"],
+  moduleNameMapper: {
+    "\\.(png|gif|jpe?g|svg)$": "<rootDir>/test/__mocks__/fileMock.js",
+    styles: "identity-obj-proxy"
+  },
+  transformIgnorePatterns: ["node_modules/(?!cozy-ui)"]
```

Comment:
Could you push this in another Pull Request?

Reply:
Done here : https://github.com/cozy/cozy-contacts/pull/34